src/components/pages/telling-story/hero/form/form.jsx

```diff
@@ -0,0 +1,207 @@
+          <Field
+            fieldName="Cilium Slack Username"
+            type="text"
+            name="ciliumSlackUsername"
+            {...register('ciliumSlackUsername')}
+          />
+          <Field
+            fieldName="How are you using Cilium?"
+            type="text"
+            name="howAreYouUsingCilium"
+            {...register('howAreYouUsingCilium')}
+          />
+        </div>
+        <Field tag={FIELD_TAGS.TEXTAREA} fieldName="Message" {...register('message')} />
+        <div className="border-b border-gray-4 pb-4">
+          <span className="text-sm font-semibold">What do you need help with?</span>
+          <div className="mt-5 gap-x-8 grid md:grid-cols-[repeat(2,minmax(45%,max-content))] gap-y-4">
+            <Checkbox
+              id="reviewAbstract"
+              label="Reviewing an abstract"
+              name="reviewAbstract"
+              value="reviewAbstract"
+              {...register('reviewAbstract')}
+            />
+
+            <Checkbox
+              id="writePost"
+              label="Writing a blog post"
+              name="writePost"
+              value="writePost"
+              {...register('writePost')}
+            />
+
+            <Checkbox
+              id="polishPresentation"
+              label="Polish a presentation"
+              name="polishPresentation"
+              value="polishPresentation"
+              {...register('polishPresentation')}
+            />
+
+            <Checkbox
+              id="getRetweet"
+              label="Getting a Retweet"
+              name="getRetweet"
+              value="getRetweet"
+              {...register('getRetweet')}
+            />
+
+            <Checkbox
+              id="findSpeaker"
+              label="Finding a speaker for an event or livestream"
+              name="findSpeaker"
+              value="findSpeaker"
+              {...register('findSpeaker')}
+            />
+
+            <Checkbox
+              id="echoNews"
+              label="Submitting to eCHO News"
+              name="echoNews"
+              value="echoNews"
+              {...register('echoNews')}
+            />
+
+            <Checkbox id="other" label="Other" name="other" value="other" {...register('other')} />
+          </div>
+        </div>
+        <Button
+          className="w-full md:w-auto"
+          size="md"
+          theme="primary"
+          type="submit"
+          loading={isLoading}
+        >
+          Send Story
+        </Button>
+      </motion.form>
+      {formState === FORM_STATES.SUCCESS && (
+        <motion.div
+          className="flex flex-col items-center justify-center"
+          initial={{ opacity: 0 }}
+          animate={{
+            opacity: 1,
+            transition: { delay: APPEAR_AND_EXIT_ANIMATION_DURATION },
+          }}
+        >
+          <img src={SuccessHero} alt="" loading="eager" />
```

Comment:
Since Success hero is not a component, it should be written with a lowercase letter.